main.cpp

```diff
@@ -28,7 +28,19 @@ int main(){
       cout << "=" << num1*num2 << endl;
     }
     else if( input == "divide" ){
-      cout << "=" << num1/num2 << endl;
+      int remainder = num1%num2;
```

Comment:
Move this into the beginning of the else, above the first cout statement, we don't want to divide before we check if the num2 is 0.